tests/test_PETSc.py

```diff
@@ -68,12 +68,14 @@ def test_ldc():
     parameters = {"Reynolds Number": 100}
     n = nx * ny * nz * dof
 
+    # FIXME why not state = numpy.arange(1, n+1) ?
```

Comment:
Can you please remove the intermediate comments and disabled tests? Presumably they still pass in serial mode.